tests/tt_metal/tt_metal/test_kernels/misc/watcher_waypoints.cpp

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+#include "ckernel.h"
+#include "debug/status.h"
+
+// Wait cycles to create a ~1sec delay
+#define WAIT_CYCLES 1200000000
```

Comment:
I know I'm contradicting myself, but we can grab this on the host through this API:
int tt_SiliconDevice::get_clock(int logical_device_id)
Almeet may know how to plumb this

Reply:
Fixed this in latest commit